testsuite/dmtf_mof_schema_def.py

```diff
@@ -55,16 +44,9 @@
 # This should not change unless you intend to use another schema directory
 SCRIPT_DIR = os.path.dirname(__file__)
```

Comment:
The name "SCRIPT_DIR" may have made sense earlier, but now this is the test directory.

Reply:
TODO